Library/Homebrew/dev-cmd/man.rb

```diff
@@ -18,6 +18,10 @@ def man
     else
       regenerate_man_pages
     end
+
+    git_diff_args = ["-C", HOMEBREW_REPOSITORY, "diff", "--stat"]
+    git_diff_args << "--exit-code" if ARGV.flag? "--exit-code"
+    exit system "git", *git_diff_args, "docs/brew.1.html", "manpages"
```

Comment:
I had thought it would be fun to display a `git diff --stat` whenever `brew man` was invoked, so you could see if there are changes:

~~~
$ brew man
Writing HTML fragment to /usr/local/Homebrew/docs/brew.1.html
Writing man page to /usr/local/Homebrew/manpages/brew.1
Writing man page to /usr/local/Homebrew/manpages/brew-cask.1
 docs/brew.1.html | 12 +++++++++++-
 manpages/brew.1  | 11 ++++++++++-
 2 files changed, 21 insertions(+), 2 deletions(-)
~~~

If we only want that to be displayed when `--exit-code` is passed, then I will wrap those commands in an `if` block as you suggest.